gen/dcos-config.yaml

```diff
@@ -1449,7 +1449,7 @@ package:
       # Read metrics from Admin Router Nginx Prometheus endpoint.
       [[inputs.prometheus]]
         ## An array of urls to scrape metrics from.
-        urls = ["https://localhost/nginx/metrics"]
+        urls = ["https://127.0.0.1/nginx/metrics"]
```

Comment:
Since this is a HTTPS URL, we should be validating the TLS connection. At the very least, can we add a JIRA issue and a comment here that this should be changed.

Reply:
Hey Jon, I've created a follow-up ticket for this:
https://jira.mesosphere.com/browse/DCOS_OSS-4578